src/main/java/mate/academy/hibernate/relations/model/Country.java

```diff
@@ -5,7 +5,7 @@
 import javax.persistence.GenerationType;
 import javax.persistence.Id;
 
-@Entity(name = "countries")
+@Entity
```

Comment:
```suggestion
@Entity
@Table(name = "countries")
```
it's still better to call tables, which store info about multiple entities, in plural